README.md

```diff
@@ -93,6 +93,21 @@ $ grunt prod
 
 * explore `config/env/production.js` for production environment configuration options
 
+### Running with User Seed
+To have an 'admin' and 'user' user account seeded at runtime:
```

Comment:
I agree with the correction there for the admin account, though let's make sure we also specify the actual username used for seeding so it's not ambiguous.